daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestBlob.scala

```diff
@@ -110,19 +111,23 @@ class TestBlob {
    *
    ***/
   @Test def test_2GB_blob(): Unit = {
-    val schema = path("daffodil-cli/src/test/resources/org/apache/daffodil/cli/large_blob.dfdl.xsd")
+    val schema = path(
+      "daffodil-cli/src/test/resources/org/apache/daffodil/cli/large_blob.dfdl.xsd",
+    )
```

Comment:
We have two options:

1. Increase maxColumns (which means a lot of files get reformatting changes again).
2. Shorten our file paths ("daf/cli/src/test/resources/org/apache/daffodil/cli/large_blob.dfdl.xsd" or go even further and remove the org/apache/daffodil entirely, since test files can use shorter paths).

Reply:
Or maybe define prefixes in a constant and reference that later:

```scala
val cliTestResources = "daffodil-cli/src/test/resources/org/apache/daffodil/cli"

val schema = path(s"{cliTestResources}/large_blob.dfdl.xsd"`)
```

Reply:
One reason I like the full path is it's easy to copy/paste when a test fails and I need to manually run a test. But I guess that happens rarely enough, and it's not much extra effort to manually replace the variable with the common part.

This seems like a reasonable approach. I guess it's also good that it removes more boilerplate, and makes the actual test file (i.e. the important thing) stand out a bit more.

Reply:
Using a constant to shorten paths is reasonable as well although I've used copy/paste on test file paths to go look at a test file like Steve too.  Everything we do with test files will become easier to do if we actually remove their parent directories as much as possible, though.  Many tests shouldn't need more than 1 or 2 parent directories, IMO.

Whatever we decide to do, let's do it in another PR.  I want to merge this PR before I work on my next runtime2 PR (daffodil-codegen-c and DaffodilC instead of Runtime2).  How long until we can hear from @mbeckerle too?